src/python/doc/alpha_complex_user.rst

```diff
@@ -11,8 +11,8 @@ Definition
 
 `AlphaComplex` is constructing a :doc:`SimplexTree <simplex_tree_ref>` using
 `Delaunay Triangulation  <http://doc.cgal.org/latest/Triangulation/index.html#Chapter_Triangulations>`_
-:cite:`cgal:hdj-t-19b` from `CGAL <http://www.cgal.org/>`_ (the Computational Geometry Algorithms Library
-:cite:`cgal:eb-19b`).
+:cite:`cgal:hdj-t-19b` from the `Computational Geometry Algorithms Library <http://www.cgal.org/>`_
```

Comment:
What was it a duplicate of? I thought this was the branch without the new plugin, so I am confused...

Reply:
sphinx was complaining:
```
alpha_complex_user.rst:6: WARNING: Duplicate explicit target name: "cgal".
```
because `'CGAL <http://www.cgal.org/>'_` is also creating a cgal target.

Reply:
Yes, but where is the **other** one? Is it the one in installation.rst? Ah, I guess this is the same problem as with the bibliography, everything is global... I am starting to understand, slowly.

Reply:
Yes that's it. Everything is global, but you can't reach the link...
At least, when you use `sphinx.ext.autosectionlabel` it may be clearer that it is global, no ?

Reply:
But the target in installation.rst is also towards www.cgal.org (assuming it uses the explicit `` `CGAL <https://www.cgal.org/>`_`` and not the section title), so here we could just write `CGAL_` and it would find the right link without duplicating the target?

Reply:
Ah, no, the other one is from the .inc file, that makes more sense. And that one links to the installation page. Well, it wouldn't be so bad if this one also linked to installation.html#cgal, but ok, things make more sense now.